pkgs/development/cuda-modules/generic-builders/manifest.nix

```diff
@@ -40,6 +41,9 @@ let
 
   inherit (stdenv) hostPlatform;
 
+  # Override function for the package.
+  maybeOverride = overrides.${pname} or null;
```

Comment:
Now this is kind of different from `overrides.nix` in the sense that we have an attribute `cudnn_8_9`, and you'd expect that adding `overrides/cudnn_8_9.nix` would affect it but it won't.

However, exactly because the "cudnn.nix" "fixup" is applied to all instances of "cudnn" coming from manifests, the semantics here is different from "just apply overrides to attributes in the very last step of composition"

Reply:
Alright second attempt at interpretation: "fixups" are "early overrides", in the sense that they are applied before we create `.override`